RandomizerHost/ViewModels/MainWindowViewModel.cs

```diff
@@ -160,8 +170,8 @@ public async void CreateFromRandomSeed(Window in_Window)
         {
             try
             {
-                this.PerformRandomization();
-                this.RandoSettings.SeedString = RandomMM2.Seed.SeedString;
+                this.PerformRandomization(in_DefaultSeed: true);
```

Comment:
I'm not familiar with this syntax. Is this passing a value (`true`) for an argument by name (`in_DefaultSeed`)?

Reply:
Yes, it is a way to specify the argument you are filling in a method call.  I did this here because it is not really obvious what the "true" means when we call this method without actually going to look at the method signature.  I think this makes it more understandable by someone reading the code.  Otherwise, you just see a random boolean constant being passed in, and might wonder what it is for.